packaging/ubuntu-14.04/snapd.postinst

```diff
@@ -20,4 +20,6 @@ case "$1" in
             systemctl -f enable snapd.autoimport.service snapd.socket snapd.service snap.mount.service snapd.system-shutdown.service
             systemctl start snapd.autoimport.service snapd.socket snapd.service snap.mount.service
         fi
+
+	echo "Please reboot, logout/login or source /etc/profile to have /snap/bin added to PATH."
```

Comment:
Would it make sense to do a `echo $PATH | grep -q /snap/bin` here so that this is not displayed on systems that already had a previous version of snapd?